code/modules/power/engines/supermatter/supermatter.dm

```diff
@@ -549,13 +557,9 @@
 		if(power_changes)
 			power = max((removed.temperature() * temp_factor / T0C) * gasmix_power_ratio + power, 0)
 
+		gas_coefficient = 1 + (combined_gas / MOLE_CRUNCH_THRESHOLD) * (plasmacomp * PLASMA_CRUNCH + o2comp * O2_CRUNCH + co2comp * CO2_CRUNCH + n2comp * N2_CRUNCH + n2ocomp * N2O_CRUNCH)
 		if(prob(50))
-			var/mole_crunch_bonus = 0
-			if(combined_gas > MOLE_CRUNCH_THRESHOLD)
-				mole_crunch_bonus = 7000 //This adds 7000 EER worth of power to the SM. This should make mole crunch potentially worthy as a SM setup, if not risky. More stable than the anomalies, but very easy to push over the edge. Don't forget, a high EER setup can harvest power through zaps
-				if(!has_been_powered) // Nice try, no free no risk power
-					enable_for_the_first_time()
```

Comment:
As all power generation is a function of EER, and 0*anything remains zero, we are good to remove this. If someone finds a way to break it later, it should be easy enough to implement a check.